src/garage/torch/algos/vpg.py

```diff
@@ -0,0 +1,286 @@
+"""Vanilla Policy Gradient."""
+import collections
+
+from dowel import tabular
+import numpy as np
+import torch
+import torch.nn.functional as F  # noqa
+
+from garage.misc import special
+from garage.np.algos import RLAlgorithm
+from garage.tf.misc import tensor_utils
+
+
+class VPG(RLAlgorithm):
+    """Vanilla Policy Gradient.
```

Comment:
It would be good to mention that this algorithm is also known as REINFORCE.

Reply:
fixed

Reply:
I don't see the term REINFORCE anywhere in this docstring

Reply:
@yonghyuc please address this comment...

Reply:
sorry, 
I just added "REINFORCE" word in the docstring